cmd/kritis/kubectl/plugins/resolve/plugin.yaml

```diff
@@ -1,3 +1,8 @@
 name: "resolve-tags"
 shortDesc: "resolve images in kubernetes yamls from tags to digests"
-command: "./resolve-tags"
+command: "./resolve-tags $PWD -f=$KUBECTL_PLUGINS_LOCAL_FLAG_FILENAME"
```

Comment:
Sorry, i dont have much context, but from what i heard yesterday, you can't pass multiple args flags?
How will users use this plugin?
```
export KUBECTL_PLUGINS_LOCAL_FLAG_FILENAME=myfile.yaml
kubectl resolve-tags
```
or
```
kubectl resolve-tags -f myfile.yaml
```

Reply:
Can we also add a short README with usage in plugins/resolve/README ?